packages/ember-runtime/lib/system/core_object.js

```diff
@@ -132,11 +132,7 @@ function makeCtor() {
               let baseValue = this[keyName];
 
               if (baseValue) {
-                if ('function' === typeof baseValue.concat) {
```

Comment:
removed brittle way of checking if array, `makeArray` already contains proper `isArray` check